service/storage_service.cc

```diff
@@ -3572,6 +3575,252 @@ future<> storage_service::node_ops_abort_thread() {
     });
 }
 
+class random_mutation_generator {
```

Comment:
We already have a random data generator that is much more featureful, see https://github.com/scylladb/scylla/blob/master/test/lib/random_schema.hh#L97

Reply:
Actually this is the end-user function: https://github.com/scylladb/scylla/blob/29a674cd9400b460a48cbf8ab4a362a939a926ca/test/lib/random_schema.hh#L250.
It needs a random schema instance, but you can easily add a `random_schema` constructor which takes an already constructed schema object, if you don't want a random schema.

Reply:
Thanks. We can definitely try to reuse some of the code. But for now, let's first discuss if this feature is useful.